client/src/UnconfiguredDashboard.js

```diff
@@ -1,33 +1,28 @@
 import React from 'react';
 
-import { DashboardLogo } from 'flight-webapp-components';
-
 function UnconfiguredDashboard() {
   return (
     <div>
-      <DashboardLogo />
       <p>
-        The Flight File Manager Service allows you to manage your files on
-        your cluster in a familiar file browser GUI (graphical user interface)
-        from within the comfort of your browser.
+        Flight Console Service allows you to access an interactive terminal
+        console session running on your cluster from the comfort of your
+        browser.
       </p>
 
       <p>
-        Before Flight File Manager Service can be used, it needs to be
-        configured by your system administrator.  It can be configured by
-        running:
+        Before Flight Console Service can be used, it needs to be configured
```

Comment:
Swap 'Console' for 'Files', here and above :)

I would also remove the word 'Service'. I don't think the apps are referred to in that way anywhere else